hadoop-hdds/tools/pom.xml

```diff
@@ -54,6 +54,12 @@ https://maven.apache.org/xsd/maven-4.0.0.xsd">
       <artifactId>ratis-tools</artifactId>
       <groupId>org.apache.ratis</groupId>
       <version>${ratis.version}</version>
+      <exclusions>
```

Comment:
I would fix it from the Ratis side: RATIS-827

Reply:
I had seen that issue.  The reason I included it here as a workaround is there were warnings related to it during shading.

Reply:
Make sense to me. I've +1 on RATIS-827. We can backoff this change when updating the ratis-snapshot version.

Reply:
We can remove it later, but I don't think it's strictly necessary.  Ozone builds fine using locally built Ratis snapshot with RATIS-827 applied.

Reply:
Sure, I agree. We can remove the exclusion in the commit where we start to use a Ratis version which includes RATIS-827. It was just a quick reminder that we can remove it after RATIS-827.